astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -140,7 +166,7 @@ def execute(self, context: Context) -> None:
                 aws_conn_id=self.aws_conn_id,
                 target_states=self.target_states,
                 failed_states=self.failed_states,
-                poll_interval=self.poll_interval,
+                poll_interval=self.poke_interval,
```

Comment:
```suggestion
                poke_interval=self.poke_interval,
```

Reply:
Ignore EMR and dbt sensor renaming of `poll_interval` in this PR, because sensor triggers inherits from the operator triggers class, so it will covered in the next PR which can cover the renaming of `poll_interval` to `poke_interval`